06-building-a-program-start-to-finish/writing-tests/output/inverted-index-test.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable */
+const books = require('../books.json');
+const adventureBooks = require('../adventure-books.json');
+const emptyArray = require('../empty.json');
+const invalidContent = require('../invalid-content.json');
+const invalidFile = require('../invalid-file.json');
+const invalidKey = require('../invalid-key.json');
+/* eslint-enable */
+
+const index = new InvertedIndex();
+
+describe('Test for valid book data', () => {
```

Comment:
Consult with your team mates on how to describe test suites and specs using BDD.